### PR TITLE
Improve wording in postman_collection.md

### PR DESCRIPTION
**Pull request by dichotommy**
Anglicize some French-sounding phrases, smooth out the English wording and make it consistent.

### CODE REVIEW

File: create/how_to/postman_collection.md [hunk 1]
```diff
@@ -1,30 +1,29 @@
 # Postman Collection for MeiliSearch
 
-Are you tired to use the `curl` command in your terminal to test MeiliSearch? It can be tedious to re-write every route when wanting to try out an API.
+Are you tired of using the `curl` command in your terminal to test MeiliSearch? It can be tedious to re-write every route when wanting to try out an API.
 
-Postman is a platform that lets you create HTTP requests that you can reuse easily and share with everyone.
-We provide a [Postman collection](/postman/meilisearch-collection.json) containing all the routes of the MeiliSearch API! 🚀
+Postman is a platform that lets you create HTTP requests that you can reuse easily and share with everyone. We provide a [Postman collection](/postman/meilisearch-collection.json) containing all the routes of the MeiliSearch API! 🚀
```
Comment (guimachiavelli) [inline create/how_to/postman_collection.md:5]:
```suggestion
Postman is a platform that lets you create HTTP requests you can easily reuse and share with everyone. We provide a [Postman collection](/postman/meilisearch-collection.json) containing all the routes of the MeiliSearch API! 🚀
```

**Review by guimachiavelli — APPROVED** [review]
🚀

**Comment by dichotommy** [comment]
bors try

**Comment by dichotommy** [comment]
bors merge

**Review by guimachiavelli — APPROVED** [review]

**Comment by dichotommy** [comment]
bors merge

**Pull request merged**